electricpy/visu.py

```diff
@@ -336,62 +336,62 @@ class PowerCircle:
 
     Parameters
     ----------
-    type: str "sending" or "receiving"
-
-    Vr: complex Receiving End Voltage
-
-    Vs: complex Sending End Voltage
-
-    power_factor: float Power Factor
-
-    Pr: float Receiving End Real Power
-
-    Qr: float Receiving End Reactive Power
-
-    Sr: float Receiving End Total Power
-
-    Ps: float Sending End Real Power
-
-    Qs: float Sending End Reactive Power
-
-    Ss: float Sending End Total Power
-
-    power_factor: float Power Factor
-
-    A: float ABCD Matrix
-    
-    B: float ABCD Matrix
-    
-    C: float ABCD Matrix
-    
-    D: float ABCD Matrix
-
+    power_circle_type: str ["sending", "receiving"]
+    Vr: complex 
```

Comment:
In reference to the [discussion here](https://github.com/engineerjoe440/ElectricPy/pull/36#discussion_r781905908), I see the confusion. There isn't a very *clean* or *useful* way to explain to users that some options are exclusive. There's a couple of things that we can do...

1. Perhaps the simplest, but least effective in communicating to users is simply to add documentation to each of the exclusive inputs; something like: "`Receiving End Voltage. This argument should only be specified in the case where Vs is NOT used; the two arguments are exclusive.`"  Clearly it spells out what the intention is, but doesn't really help a user understand their error when they're working on their final research project at 2AM.
2. Another option is to make specific objects that are responsible for each behavioral case. That means you could still keep this class, but just wrap it with functions to simplify the usage. Think of having a `sending_power_circle` function and a `receiving_power_circle` function. It's a little more overhead, admittedly, but it helps with the clarity of the usage, and would allow you simplify the interface for users. It's also a pretty clear delineation for users to understand which function they should use. Again, if you were to go this route (and it's what I'd suggest), I'd aim to reduce the input list in the functions to only what they need. You would still use this class as the "core" of the functionality, the `sending/receiving_power_circle` functions would just wrap it to make using everything easier.

Reply:
I have made changes accordingly by adding functions `sending_end_power_circle` and `receiving_end_power_circle`, I am not very sure how they would work even though I have tested them with a test case from a stander textbook problem, 

Sir, it up to you whether to include this function or not ,

Thank you